### ISSUE TITLE
Feature: Inbox By Google 

### ISSUE THREAD

**Issue by NocturneBlock** [inbasic/ignotifier#304]
Google has a new service called Inbox, which is a variant inbox for Gmail, accessed via inbox.google.com. It would be nice if the option of using this in place of the default Gmail page was available in Notifier.

**Comment by ocombe** [comment]
Yes please, it would be cool if this addon could open inbox instead of gmail :)

**Comment by lililya** [comment]
thank

**Comment by lazydevorg** [comment]
+1

**Comment by inbasic** [comment]
Well, switching to inbox.google.com is not that easy. For instance, if you click on the title of an email, right now the thread related to that conversion is opened. However, in inbox.google.com, threads do not have different ids. So how the extension is supposed to open the threads?

**Comment by Kougeru** [comment]
Coming here to +1 it but you have a very good point. I'm not a programmer so the only answer for that I have is to ask Google how to do it lol. Is there no API out for Inbox yet? I love Inbox far more than the current Google though. It boggles my mind why it's a whole new thing rather than a Gmail UI update.

**Comment by lililya** [comment]
in box I had it, but the mailbox is not the same.Need to look at what it feels like, and if you have to leave traces login ......

**Comment by lililya** [comment]
不想提谷歌，我们国家限制了我很久没上。居然有人使用我的号，还挂我照片用我好姐妹的名字。我以为中国出了个好厉害的

**Comment by lililya** [comment]
我以为中国有个好厉害的设计师。叫linqiao,居然使用我账户，躺着都中枪。我是不会画图，不会设计加英语烂。。。

**Comment by cautionbug** [comment]
Sent an email to @inbasic before i found this feature request... i'm sure eventually Google will have to release an API for Inbox, but currently there isn't one. 

i did find this when inspecting an email item with Firebug (actual identifiers scrubbed):
```html
<div 
    jsan="X.xx,x-XXXXXXXXX,X.scroll-list-item,7.top-level-item,7.scroll-list-item-open,0.data-item-id,0.data-item-id-qs,0.tabindex,0.jsaction" 
    style="" 
    class="bw scroll-list-item top-level-item YqfB2e scroll-list-item-open" 
    jstcache="169" 
    jsinstance="*0;#gmail:thread-f:##############" 
    data-item-id="#gmail:thread-f:##############" 
    data-item-id-qs="qs-gmail-thread-f-##############-0" 
    tabindex="0" jsaction="none">
```
i have no idea if or how the `jsinstance`/`data-item-id` attributes or `#gmail:thread` hashes could be used to bring up a thread directly, but maybe it can be useful?

Leave it to Google to produce something ALMOST as useful as what you had, but get people clamoring to use it anyway.

**Comment by inbasic** [comment]
@cautionbug even if we manage to find these variables still it is not possible to use them to force inbox open a thread. Currently, GN generates a uril something like `https://mail.google.com/mail/u/0/?shva=1#inbox/#########` to open a thread in a new tab, however, in inbox, you need to open the main UI then somehow force a thread to be opened.

**Comment by cautionbug** [comment]
Yeah, i kinda figured it wasn't much of an option at this point. What about
a separate button (temporarily) next to the G button to open Inbox instead?
If it's better just to wait for an API, i totally understand. :-)

**Comment by Keiichii** [comment]
"What about a separate button (temporarily) next to the G button to open Inbox instead?"
+1!!!! I really would like it!
Or just an option in plugin's settings, to choose a button "gmail/inbox"
Thanks!
By the way, the best plugin! ;)

**Comment by zehevi** [comment]
Go to your Inbox > Settings > Other
There you can check "Redirect Gmail to Inbox.google.com"
Everytime you attempt getting to Gmail you'll end up in Inbox.

**Comment by drazisil** [comment]
Thanks to @zehevi for that tip.

I came here to :+1: this request as well, with mailbox shutting down I guess I have to switch to inbox.
